tests/test_decorators.py

```diff
@@ -54,6 +54,21 @@ def foobar(text):
         assert 'report:/tmp' in str(e)
 
 
+def test_decorator_with_type_exception():
+    """Ensure that the library handles non-subclassed exceptions"""
+
+    @exception_report()
+    def foobar(text):
+        raise TypeError("bad things!!")
+
+    try:
+        foobar('hi')
+    except Exception as e:
+        if 'bad things' not in repr(e):
+            raise
+        assert 'report:/tmp' in str(e)
```

Comment:
minor: why not use pytest.raises context manager?

Reply:
good question. short answer is that I copy pasted from other tests.  investigating though.

Reply:
switched it to use pytest.raises